app/components/admin/participants/validation_tasks.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2 class="govuk-heading-m govuk-!-margin-top-9">Validation tasks</h2>
+
+<ol class="govuk-list govuk-list--number">
+  <% validation_steps.each do |step| %>
+    <li>
+      <table data-test="admin-participants-validation-tasks" class="govuk-table">
+        <thead  class="nhsuk-table__head">
+          <tr class="govuk-table__row">
+            <th class="govuk-table__header" colspan="2"><%= t :name, scope: "schools.participants.validations.#{profile.participant_type}.#{step}" %></td>
+          </tr>
+        </thead>
+        <tbody>
+          <tr class="govuk-table__row">
+            <td class="govuk-!-width-full">
+              <%= govuk_link_to(
+                    t(:link_text, scope: "schools.participants.validations.#{profile.participant_type}.#{step}"),
+                    validation_step_admin_participant_path(profile, step)
+                  )
+              %>
+            </td>
+            <td>
+              <%= govuk_tag tag_attributes(step) %>
+            </td>
```

Comment:
Does putting it in a table (rather than in a flexbox of our own making) cause the tag to break over multiple lines for unlucky viewport sizes?

Reply:
IIRC html tables never wraps - but it might just happily go out of the screen.